ibm/mas_devops/roles/cp4d_wds/defaults/main.yml

```diff
@@ -0,0 +1,22 @@
+# Configure Discovery Channel and Version info
+# If not specified, for wdschannel it's v4.0 by default , and for wdsversion it's 4.0.4 by default
+wdschannel: "{{ lookup('env', 'WDS_CHANNEL') }}"
```

Comment:
the defaults need to be set on the lookup. See ither default/main.yml files for examples.

Reply:
fixed.